frontend/src/components/print/__tests__/repairPrintConfig.spec.js

```diff
@@ -817,14 +817,14 @@ describe('repairConfig', () => {
     })
   })
 
-  describe('story', () => {
+  describe('summary', () => {
```

Comment:
If you wanted to make it perfect, you would need to test that
the people, who had story in their  local storage print config, now have summary with contentType 'StoryContext' in their local storage print config.

But i think it is no problem that all people loose their config for this deployment.

Reply:
I thought about that, but I didn't want to convert from story to summary.